src/GraphQL/CacheableGraphqlValue.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * @file
```

Comment:
That should be the class documentation header and not (at)file. Let's put that above the class instead.